### DISCUSSION TITLE
Difficulty dropdown isn't visible at the bottom of beatmap listing

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#16119] [Q&A] [answered]
When you search for specific beatmaps and the amount of items is limited, difficulty dropdown of a beatmap card is covered by the end of page.

https://user-images.githubusercontent.com/53346335/146408971-8b1987e4-5794-4a89-b95a-da133a1b364e.mp4

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #16120
